sandbox/controls/ListItems.qml

```diff
@@ -357,6 +368,8 @@ CExPynn1gWf9bx498P7/nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2I
                       nzPcxEzGExhBdJGYihtAYQlO+tUZvqrPbqeudo5iJGEJjCE15a3VtodH3q2ImYgiNITTlTdG1nUZ5a92VITQxITFiJmIIjSE0htAYQrMHAAD//+wwFVpz+yqXAAAAAElFTkSuQmCC"
         image.isIdenticon: true
         isOnline: true
+        statusListItemIcon.identiconRing.ringSpec: identiconRing1
+        statusListItemIcon.identiconRing.totalRingUnits: 25
```

Comment:
just for the sake of beautifying: I imagine that `statusListItemIcon` is part of `StatusListItem` and not of `StatusMemberListItem` because of Mouse events and z order reasons? I was wondering, is there any case we could avoid 'dotdotting' like `statusListItemIcon.identiconRing.totalRingUnits:` and have something more straight forward like `totalRingUnits:` instead?

Reply:
Yes, I agree with your point of view, however... I don't know which is the best point to expose these properties but giving as much flexibility as possible to the component. `StatusListItem` is a super flexible component that exposes, I don't know if all, but most of its components and allows you to design a super custom list item. 

I'm open to proposals! Maybe it could have sense to add all `StatusIdenticonRing`properties as well in `StatusSmartIdenticon` so in this particular case we'd have something like this:

- `statusListItemIcon.ringSpecs` or `statusListItemIcon.totalRingUnits`

WDYT?

Reply:
Ok! It sound like a good idea. Going for that! :)